docs/troubleshooting-guide.md

```diff
@@ -183,8 +183,12 @@ Furthermore, you might see tags such as `${test.student1}` and `${version}` in s
 
 1. **ERROR (on Linux)**: `java.io.IOException: Directory "/tmpfiles" could not be created`.
 
-   **SOLUTION**: Add `-Djava.io.tmpdir=/path/to/teammates/tmp` for the tests' run configurations. The "tmp" folder in the specified directory needs to be created before running the tests.
+  **SOLUTION**: Add `-Djava.io.tmpdir=/path/to/teammates/tmp` for the tests' run configurations. The "tmp" folder in the specified directory needs to be created before running the tests.
```

Comment:
Add a single space before "**SOLUTION**..." so that you don't change this line that is unrelated to the current issue.